agir/front/components/genericComponents/EventCard.js

```diff
@@ -147,7 +147,8 @@ const StyledCard = styled(Card)`
 
   ${Row} {
     font-size: 0.875rem;
-    max-width: calc(100% - 2rem);
+    max-width: 106%;
+    display: block;
```

Comment:
C'est un peu bizarre d'utiliser `106%`, ça pourrait peut-être marcher :
```suggestion
    max-width: calc(100% + 2rem);
```